### ISSUE TITLE
Add Option to Download Payslip

### ISSUE THREAD

**Issue by zoraaisd** [zoraaisd/hrms#58]
DESCRIPTION :
Currently, the HRMS application does not provide an option for employees to download their payslips. A download feature should be added for better accessibility and record-keeping.

STEPS TO REPRODUCE:
1.Navigate to the website
2.Login into employee
3. click on payslip
4.Generate payslip
5. Observe that there is no download option for generated payslip

EXPECTED RESULT :
Provide a Download button in the Payslip section
Payslip should be downloadable in formats like PDF
File should contain accurate salary details
File name should include employee name and month

ACTUAL RESULT :
Payslip is only viewable on screen and cannot be downloaded.

SEVERITY : Minor

PRIORITY:P2

REPORTER : Priya

SCREENSHOT :

<img width="1053" height="537" alt="Image" src="https://github.com/user-attachments/assets/28ccf28a-6a23-4436-a76e-df9f88296ac0" />
